gensim/corpora/textcorpus.py

```diff
@@ -44,93 +44,13 @@
 
 from gensim import interfaces, utils
 from gensim.corpora.dictionary import Dictionary
-from gensim.parsing.preprocessing import STOPWORDS, RE_WHITESPACE
+from gensim.parsing.preprocessing import remove_stopword_tokens, remove_short_tokens, \
```

Comment:
Please use this style of line continuation:

```suggestion
from gensim.parsing.preprocessing import (
    remove_stopword_tokens, remove_short_tokens, …,
    …,
)
```

Reply:
Done